web-registry/src/pages/BasketDetails/BasketDetails.tsx

```diff
@@ -11,9 +18,45 @@ const BasketDetails: React.FC = () => {
   return (
     <>
       {data.overview && <BasketOverview {...data.overview} />}
-      {data.creditBatches && <BasketEcocredits batches={data.creditBatches} />}
+      {data.creditBatches && (
+        <SectionLayout>
+          <BasketEcocreditsTable batches={data.creditBatches} />
+        </SectionLayout>
+      )}
     </>
   );
 };
 
 export { BasketDetails };
+
+// The following component is of type layout, something
+// provisional since this section will contain tabs soon
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
@haveanicedavid recently [mentionned](https://github.com/regen-network/regen-web/pull/1034#discussion_r915176293) to me that `useStyles` is deprecated in MUI and that we should avoid using it from now on.

Reply:
@flagrede I do think we should try to do that moving forward especially where it's an easy switch because we'll have to eventually get rid of all `makeStyles` to upgrade to react 18, but unfortunately while the `<Section />` component has `sx` values for both root and title, they aren't configured in a way that will properly allow using padding in this way (IE using the `theme` callback). So these styles would need to be reworked, or the section component would need be modified, which is probably more work than it's worth given this story is blocking others

Reply:
Ok, let's leave it for a later refactor then.